### ISSUE TITLE
Builds "failing" with strange docker error but actually passes

### ISSUE THREAD

**Issue by faceleg** [lambci/ecs#9]
I'm getting builds consistently failing with one of the errors:

`Error response from daemon: You cannot remove a running container <hash>. Stop the container before attempting removal or use -f`

`write /dev/stdout: resource temporarily unavailable`

The build actually continues and an artefact is deployed.

What could cause this? What might I try to start debugging/fixing this?

(for my own reference, examples:
`KS74D3U6ABIOa54e258f786ed94b6fc8ec2b66a41a75de6d62b95ec49f41ab59920b95429203`
`3deb346072699d4a24fbf52848a1183e620e64ce4ccd6a0b61cd8dceca651c20`)

**Comment by mhart** [comment]
@faceleg is one of your builds still running on that machine? It could be this command that's failing: https://github.com/lambci/ecs/blob/master/runbuild.sh#L20 (basically, the `--rm` flag is trying to remove any existing containers with the same name – but if one of them is still running, then I think it'll fail)

**Comment by faceleg** [comment]
Yes I think this was the case. I killed the EC2 instance, which caused ECS to spawn another. Builds have been passing successfully since then.

**Comment by faceleg** [comment]
I've been experiencing this again. I have deleted the ECS cluster and rebuild. First build was good. second build failed with

`Error response from daemon: You cannot remove a running container be7faf28c4431951449975645bd4c1550b5aaef2bb1aa61b63f1d1466ff2246d. Stop the container before attempting removal or use -f`

**Comment by ancil26** [comment]
We're getting the same error after restarting the ecs cluster @mhart , 

`Error response from daemon: You cannot remove a running container d4a274e5961b7b0c37620cd2f490ca37a3dcd65ffb4dd567c64bd5e03d3f8d7c. Stop the container before attempting removal or use -f`
